bot/bot.go

```diff
@@ -48,12 +49,11 @@ func NewAIBot(botCtx context.Context, aiClient *gpt.Client, discordSession *disc
 	err = json.Unmarshal(promptBytes, &promptMessages)
 
 	if err != nil {
-		logger.Panic("Failed to parse initial prompt", zap.Error(err))
+		log.Panic("Failed to parse initial prompt", err)
 	}
 
 	bot := &AIBot{
 		discordSession: discordSession,
-		logger:         logger,
```

Comment:
Now that it's not a third party package, I feel a lot safer just calling `slog.Default()` to get the logger as needed, rather than store an instance of the logger on a bunch of types